std/math.d

```diff
@@ -1419,9 +1425,11 @@ public:
  *  Mathematically, acosh(x) = log(x + sqrt( x*x - 1))
  *
  * $(TABLE_DOMRG
- *  $(DOMAIN 1..$(INFIN))
- *  $(RANGE  1 .. log(real.max), $(INFIN)) )
- *      $(TABLE_SV
+ *    $(DOMAIN 1..$(INFIN)),
+ *    $(RANGE  0..$(INFIN))
```

Comment:
Yes, sorry I should have stated this in the PR description.
See e.g. http://www.wolframalpha.com/input/?i=acosh(x)

Also `RANGE` was never defined ...